public/javascripts/weekly-review.js

```diff
@@ -0,0 +1,27 @@
+openModalButton = event => {
+  event.preventDefault()
+  fetch( '/summary', fetchParams( 'GET' ) )
+    .then( result => result.json() )
+    .then( data => populateWeeklyData( data ))
+  $( '.weekly-summary-modal' ).show()
```

Comment:
Just noticed this - shouldn't this `show` be in the promise composition? Otherwise we could be showing the modal without having completed the load...